src/gui/modalMenu.h

```diff
@@ -102,11 +116,11 @@ class GUIModalMenu : public gui::IGUIElement
 	// wants to launch other menus
 	bool m_allow_focus_removal = false;
 
-#ifdef HAVE_TOUCHSCREENGUI
-	irr_ptr<gui::IGUIElement> m_hovered;
+	// Stuff related to touchscreen input
```

Comment:
I think newlines  around this TS part should be adjusted.

Reply:
I intentionally added a blank line after the comment to make it clear that the comment refers not only to `m_touch_hovered` directly below, but also to `simulateMouseEvent`, `enter` and `leave`.